userprofile/models.py

```diff
@@ -147,15 +147,9 @@ class HealthCheckUserProfile(
     ExportModelOperationsMixin("healthcheck-user-profile"), models.Model
 ):
     ARM_CONTROL = "control"
-    # ARM_HEALTH_CONSEQUENCE = "health_consequence"
```

Comment:
Are these going to be removed permanently? In that case I would rather just remove them instead of commenting them out